docs/pages/docs.md

```diff
@@ -5,20 +5,58 @@ layout: default
-| ----------------------------------------------- | --------------------------------------------------------------- | :-: |
-| [sphinx](https://www.sphinx-doc.org/en/master/) | Generates documentation from reStructuredText or Markdown files | 🟢  |
-| [gitbook](https://www.gitbook.com/)             | General documentation builder; integrates with GitHub           | 🟠  |
-| [mkdocs](https://www.mkdocs.org/)               | Generates documentation from Markdown files                     | 🟠  |
-| [pdoc](https://pdoc.dev/)                       | Auto-generates API documentation from docstrings                | 🟠  |
-
-<details>
-<summary> 🟢 explanation</summary>
-Sphinx is the de-facto standard that is widely used. It is well tested, reliable and very customisable.
+With Python, as with many other languages, it's very common to automatically
+create a web page with documentation built from [docstrings] in your package's
+API. For your package to be easily reusable we recommend you build documentation
+pages and host them somewhere.
+
+If you're using GitHub, the simplest is to host your docs on [GitHub pages].
+[Here's how we've set this up][template-docs-dot-yaml] for [our template].
+
+<!-- URL used above in the blurb-->
+
+[docstrings]: https://peps.python.org/pep-0257/#what-is-a-docstring
+[GitHub pages]: https://docs.github.com/en/pages
+[our template]: https://github.com/UCL-ARC/python-tooling?tab=readme-ov-file#using-this-template
+[template-docs-dot-yaml]: https://github.com/UCL-ARC/python-tooling/blob/main/%7B%7Bcookiecutter.project_slug%7D%7D/.github/workflows/docs.yml
+
+## Documentation build tools
+
+| Name      | Short description                                                                                                                                               | 🚦  |
+| --------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------- | :-: |
+| [MkDocs]  | Generates documentation from Markdown files, with a plugin to generate API documentation. A good plugin ecosystem and balance of usability and customisability. | 🟢  |
+| [Sphinx]  | Generates documentation from reStructuredText or Markdown files, long been the de-facto standard and very widely used. Mature plugin ecosystem.                 | 🟠  |
+| [gitbook] | General documentation builder; integrates with GitHub.                                                                                                          | 🟠  |
+| [pdoc]    | Auto-generates API documentation from docstrings, beginner friendly but with less of a plugin ecosystem than others.                                            | 🟠  |
+
+<details markdown="block">
+<summary>More details about Sphinx</summary>
+
+We marginally recommend [MkDocs] over [Sphinx] due to it's ease of use,
+preference for markdown, and slightly better support of our preferred docstring
```

Comment:
What is our preferred docstring style? Again, unless I missed it (sorry), I think worth deferring discussion/choice of a docstring style to a new issue?

Reply:
P'raps: "more support for a variety of docstring styles" ...?

Reply:
Agree with having a separate discussion about docstring style and rephrasing as @samcunliffe suggested

Reply:
- [ ] ☝️